internal/pkg/build/sources/99-base.sh

```diff
@@ -29,7 +29,7 @@ else
     LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/.singularity.d/libs"
 fi
 
-if test -z "${PS1:-}"; then
+if [ -z "${PS1:-}" ]; then
```

Comment:
Please also apply a similar style to the corresponding line above.

Reply:
@DrDaveD 

Does "above" means L26?

https://github.com/apptainer/apptainer/blob/4b651ce60cc8570f68a06b37417d3f2a1516bd74/internal/pkg/build/sources/99-base.sh#L26

I changed L32 to match style with L26.

https://github.com/apptainer/apptainer/blob/4b651ce60cc8570f68a06b37417d3f2a1516bd74/internal/pkg/build/sources/99-base.sh#L32

Could you please elaborate on this?

Reply:
It is not showing up in the diff.

Reply:
I meant the `:-` part, so it would be:
```
if [ -z "${LD_LIBRARY_PATH:-}" ]; then
```

Reply:
Thank you for clarification @DrDaveD!